appveyor.yml

```diff
@@ -11,28 +11,37 @@ environment:
 #        Build                    #
 #---------------------------------#
 
-build: off
-
-configuration: Release
-
 install:
   - SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%;C:\MinGW\bin
   - copy c:\MinGW\bin\mingw32-make.exe c:\MinGW\bin\make.exe
   - copy c:\Python36-x64\python.exe c:\Python36-x64\python3.exe
   - python --version
   - make install
-
-after_test:
   - make package
+  - copy /Y C:\projects\dexbot\dist\DEXBot-gui.exe C:\projects\dexbot\installer\windows\msi\DEXBot.exe
   - '7z a DEXBot-gui-win64.zip %APPVEYOR_BUILD_FOLDER%\dist\DEXBot-gui.exe'
+  # read dexbot version from __init__.py
+  - for /f "delims== eol=# tokens=1,2" %%G in (.\dexbot\__init__.py) do set %%G=%%H
+  # set to version and replace ' with empty character
+  - set ApplicationVersion=%VERSION:'=%
```

Comment:
If you can get the version number, add it to the distribution name above as well like `DEXBot-gui-win64-VERSION_NUMBER.zip`

Reply:
Ok I will add this, shall I add the version to the installer as well?

Reply:
Do that. :+1: